src/components/TaskPreviewPane/TaskPreviewPane.tsx

```diff
@@ -31,14 +31,14 @@ export const TaskPreviewPane = () => {
             ? state.calendarReducer.selectedDate
             : defaultSelected,
     );
-    const transformY = useRef(new Animated.Value(-taskPreviewHeight)).current;
+    const transformY = useRef(new Animated.Value(windowHeight)).current;
 
```

Comment:
Had to tweak this animation a bit after changing to tab navigation for some reason, but now it works off native driver

Reply:
I'd rename `transformY` to `translateY`:
```suggestion
    const translateY = useRef(new Animated.Value(windowHeight)).current;
```
You're just sliding the task preview pane to show up underneath the calendar+header+status bar, right?

Reply:
| You're just sliding the task preview pane to show up underneath the calendar+header+status bar, right?

yup, i'll make the change